editions/tw5.com/tiddlers/system/doc-macros.tid

```diff
@@ -107,3 +107,23 @@ This is an example tiddler. See [[Table-of-Contents Macros (Examples)]].
 </$reveal>
 </$list>
 \end
+
+\define demo(text)
```

Comment:
I can and I guess me not liking neither the unnecessarily wordy names of those, nor the presentation does warrant proposing a different macro name or presentation in some PR.